### PR TITLE
Setup transition animation for partner cards

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Using ReactCSSTransitionGroup to introduce enter/leave animations

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "precss": "^3.1.2",
     "prop-types": "^15.6.2",
     "react": "^16.6.3",
+    "react-addons-css-transition-group": "^15.6.2",
```
Comment (taniaprocha) [inline package.json]: Why not try `popmotion`?

Comment (jfranciscosousa) [reply package.json]: Forgot about that, I'll check if it has support for enter/leave transitions and if has I will do it. I think they only make animations. React transition group is to detect changes in the `children` and apply `enter` and `leave` css classes to them.

**Comment by taniaprocha** [comment]
When I click on a partner, animation stops.

File: package.json (hunk 1 above)
Comment (jfranciscosousa) [reply package.json]: Did with popmotion, but they are JS only animations.

Comment (taniaprocha) [reply package.json]: Seems simple to use, what do you think? Should we use `popmotion`?

Comment (jfranciscosousa) [reply package.json]: Yes

**Pull request merged**